### ISSUE TITLE
create high contrast mode

### ISSUE THREAD

**Issue by julesparra** [gotechnica/technica-site-2021#63]
requirements:
1. alternative stylesheet for each page (home, sponsor, and donate) that override original styling
2. button on each page to toggle high contrast mode on/off

changes needed in alternative stylesheet:
- use `ms-high-contrast` CSS media feature to detect Microsoft High Contrast Mode & what version: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-ms-high-contrast
- make background colors white & update white text to black & colored text to brighter color (e.g. lavender -> neon purple)
- use black borders to define buttons/other elements

👀 [Bootstrap switch](https://getbootstrap.com/docs/5.0/forms/checks-radios/#switches) - fill should be pink & default state should be "off" (add to leftmost part of navbar, decrease side space between navbar links)

you can test on Windows with High Contrast Mode (try all options)!

🆘 [reference with comparative examples](https://blog.khanacademy.org/making-websites-work-with-windows-high-contrast-mode/)
